.editorconfig

```diff
@@ -280,3 +280,5 @@ dotnet_diagnostic.IDE2006.severity = warning
 # CA1822: Make member static
 # There is a risk of accidentally breaking an internal API that partners rely on though IVT.
 dotnet_code_quality.CA1822.api_surface = private
+
+dotnet_diagnostic.IDE0055.severity = none
```

Comment:
Will file issue to reenable this once we can once the formatting analyzer is fixed up to support collection expressions.

Reply:
This should be disabled for only a select group of files (e.g. 2-3 total files in the solution), and not for the entire solution.